includes/wikia/services/UserStatsService.class.php

```diff
@@ -96,6 +101,21 @@ public function increaseEditsCount() {
 	 * @return array
 	 */
 	public function getStats() {
+		// anons don't have edit stats
+		if ( !$this->validateUser() ) {
+			return [
+				'firstContributionTimestamp' => null,
+				'lastContributionTimestamp' => null,
+				'editcount' => 0,
+				'editcountThisWeek' => 0
+			];
+		}
+
+		// SUS-4773: Memoize user stats to remember them in the context of a single request and avoid network overhead
+		if ( isset( $this->cache[$this->wikiId][$this->userId] ) ) {
+			return $this->cache[$this->wikiId][$this->userId];
+		}
```

Comment:
Memcache client does that already if the same key is accessed multiple times (and there's no memcache update from the same request).

Reply:
👍 true, I'll remove